region/norfair/east.json

```diff
@@ -1567,7 +1714,15 @@
             "runways": [
               {
                 "length": 31,
-                "requires": null
+                "requires": [
```

Comment:
I think the runways in here might need a reset at the opposite door? The crumble blocks ruin them otherwise.

Reply:
Indeed adding simpified "reset" for now